### PR TITLE
Add Korean(ko) translation

### PR DESCRIPTION
**Pull request by potatogim**
Add Korean(ko) translation

Signed-off-by: Ji-Hyeon Gim <potatogim@gluesys.com>

### CODE REVIEW

**Review by kaakaa — COMMENTED** [review]
Thank you for your contribution @potatogim, and I'm sorry for late reply.

Your translation seems to be fine, but the format of the file seem to be different from those in other translated languages.  (e.g.: [active.ja.json](https://github.com/matterpoll/matterpoll/blob/master/assets/i18n/active.ja.json))

Did you follow the steps in [Translation process for new languages](https://github.com/matterpoll/matterpoll/blob/master/CONTRIBUTING.md#translation-process-for-new-languages) to create the file?

**Comment by potatogim** [comment]
> Thank you for your contribution @potatogim, and I'm sorry for late reply.
> 
> Your translation seems to be fine, but the format of the file seem to be different from those in other translated languages. (e.g.: [active.ja.json](https://github.com/matterpoll/matterpoll/blob/master/assets/i18n/active.ja.json))
> 
> Did you follow the steps in [Translation process for new languages](https://github.com/matterpoll/matterpoll/blob/master/CONTRIBUTING.md#translation-process-for-new-languages) to create the file?

Thank you for the review!

I have uploaded new patch to apply that :smile:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ The localization process is defined below:
 - Japanese: [@kaakaa](https://github.com/kaakaa)
 - Polish: [@burasuk](https://github.com/burasuk)
 - Spanish: [@ernestoittig](https://github.com/ernestoittig)
+- Korean: [@potatogim](https://github.com/potatogim)
```
Comment (hanzei) [inline CONTRIBUTING.md:30]: Please order that list alphabetically

File: README.md [hunk 2]
```diff
@@ -47,6 +47,7 @@ The currently supported languages are:
 - Japanese
 - Polish
 - Spanish
+- Korean
```
Comment (hanzei) [inline README.md:50]: Same here

**Review by hanzei — CHANGES_REQUESTED** [review]
Thanks for the translation :+1: 

Two small requests

File: CONTRIBUTING.md (hunk 1 above)
Comment (potatogim) [reply CONTRIBUTING.md]: Yes, I did that :smile:

**Review by hanzei — APPROVED** [review]
Awesome :rocket:

**Review by kaakaa — APPROVED** [review]
Great 👍

**Pull request merged**